src/acquisition/covid_hosp/state_daily/database.py

```diff
@@ -0,0 +1,79 @@
+    ('inpatient_beds_used_covid_coverage', int),
+    ('previous_day_admission_adult_covid_confirmed', int),
+    ('previous_day_admission_adult_covid_confirmed_coverage', int),
+    ('previous_day_admission_adult_covid_suspected', int),
+    ('previous_day_admission_adult_covid_suspected_coverage', int),
+    ('previous_day_admission_pediatric_covid_confirmed', int),
+    ('previous_day_admission_pediatric_covid_confirmed_coverage', int),
+    ('previous_day_admission_pediatric_covid_suspected', int),
+    ('previous_day_admission_pediatric_covid_suspected_coverage', int),
+    ('staffed_adult_icu_bed_occupancy', int),
+    ('staffed_adult_icu_bed_occupancy_coverage', int),
+    ('staffed_icu_adult_patients_confirmed_and_suspected_covid', int),
+    ('staffed_icu_adult_patients_confirmed_and_suspected_covid_coverage', int),
+    ('staffed_icu_adult_patients_confirmed_covid', int),
+    ('staffed_icu_adult_patients_confirmed_covid_coverage', int),
+    ('total_adult_patients_hospitalized_confirmed_and_suspected_covid', int),
+    ('total_adult_patients_hospitalized_confirmed_and_suspected_covid_coverage', int),
+    ('total_adult_patients_hospitalized_confirmed_covid', int),
+    ('total_adult_patients_hospitalized_confirmed_covid_coverage', int),
+    ('total_pediatric_patients_hospitalized_confirmed_and_suspected_covid', int),
+    ('total_pediatric_patients_hospitalized_confirmed_and_suspected_covid_coverage', int),
+    ('total_pediatric_patients_hospitalized_confirmed_covid', int),
+    ('total_pediatric_patients_hospitalized_confirmed_covid_coverage', int),
+    ('total_staffed_adult_icu_beds', int),
+    ('total_staffed_adult_icu_beds_coverage', int),
+    ('inpatient_beds_utilization', float),
+    ('inpatient_beds_utilization_coverage', int),
+    ('inpatient_beds_utilization_numerator', int),
+    ('inpatient_beds_utilization_denominator', int),
+    ('percent_of_inpatients_with_covid', float),
+    ('percent_of_inpatients_with_covid_coverage', int),
+    ('percent_of_inpatients_with_covid_numerator', int),
+    ('percent_of_inpatients_with_covid_denominator', int),
+    ('inpatient_bed_covid_utilization', float),
+    ('inpatient_bed_covid_utilization_coverage', int),
+    ('inpatient_bed_covid_utilization_numerator', int),
+    ('inpatient_bed_covid_utilization_denominator', int),
+    ('adult_icu_bed_covid_utilization', float),
+    ('adult_icu_bed_covid_utilization_coverage', int),
+    ('adult_icu_bed_covid_utilization_numerator', int),
+    ('adult_icu_bed_covid_utilization_denominator', int),
+    ('adult_icu_bed_utilization', float),
+    ('adult_icu_bed_utilization_coverage', int),
+    ('adult_icu_bed_utilization_numerator', int),
+    ('adult_icu_bed_utilization_denominator', int),
+    ('record_type', str),
+  ]
+
+  def __init__(self, *args, **kwargs):
+    super().__init__(
+        *args,
+        **kwargs,
+        table_name=Database.TABLE_NAME,
+        columns_and_types=Database.ORDERED_CSV_COLUMNS,
```

Comment:
would it make sense to pull these from `covid_hosp.state_timeseries.database.Database` instead of duplicating the strings here? that way if the column names ever change we'll only have to update one copy.

Reply:
the facility data uses the Database class as well but doesn't have the same column. May we refactor them into a separate file that just contains the column constants?

Reply:
Just remembered that they're not actually the same columns, they have one column name different -- `date` vs `reporting_cutoff_start`. Also annoyingly (though not for this case specifically), the date column is the second column in the time series file while the reporting_cutoff_start is the last column in the snapshot file.

:/